lib/decorators.ts

```diff
@@ -371,7 +371,10 @@ export function External<TFunction extends Function>(...args: any[]): ClassDecor
       return (target as any).__tsdi__.configureExternal(args, target);
     };
     (constructor as any).displayName = (target as any).name;
-    constructor.prototype = target.prototype;
+
+    Object.getOwnPropertyNames(target)
+      .filter(prop => prop !== '__tsdi__' && prop !== 'name' && prop !== 'length')
```

Comment:
it was originally not included so I thought I might ditch it.
If it doesn't matter, we could include it anyway.